kbase-extension/scss/partials/_createdObjects.scss

```diff
@@ -1,21 +0,0 @@
-.kb-created-objects {
```

Comment:
I don't know if this file is likely to be needed in the future for styling other parts of the object results, so I just left it in.